src/main.rs

```diff
@@ -157,6 +166,11 @@ fn main() -> anyhow::Result<()> {
             None => None,
         };
 
+        if args.flag_stdout {
+            println!("{}", config.to_toml()?);
+            return Ok(());
```

Comment:
yep, if somebody requested the help, exit code should 0, many programs get this wrong :t-rex: